common/taskana-common/src/main/resources/sql/db2/taskana_schema_update_5.2.0_to_5.10.0_db2.sql

```diff
@@ -0,0 +1,14 @@
+-- this script updates the TASKANA database schema from version 5.2.0 to version 5.10.0.
+SET SCHEMA %schemaName%;
+
+CREATE SEQUENCE TASKANA_SCHEMA_VERSION_ID_SEQ
+    MINVALUE 1
+    START WITH 100
+    INCREMENT BY 1 CACHE 10;
+
+ALTER TABLE TASKANA_SCHEMA_VERSION ALTER COLUMN ID DROP IDENTITY;
+
+-- The VERSION value must match the value of TaskanaEngineConfiguration.TASKANA_SCHEMA_VERSION
```

Comment:
I just stumbled into it: The correct reference would be: TaskanaEngineImpl.MINIMAL_TASKANA_SCHEMA_VERSION. Is it possible that you correct the documentation within in PR?

You checked that the change is compatible with the prior schema version?

Reply:
i change the comment to
```
-- The VERSION value must be equal or higher then the value of TaskanaEngineImpl.MINIMAL_TASKANA_SCHEMA_VERSION
```

yes the change is compatible with the prio schema versions